main.py

```diff
@@ -1,12 +1,176 @@
+        "7": "List all Videos",
+        "8": "Add Video",
+        "9": "Update Video",
+        "10": "Delete Video",
+        "11": "Select a Video",
+        "12": "Check out Video",
+        "13": "Check in Video",
+        "14": "Quit"
+        }
+
+    for choice_num in options:
+        print(f"Option {choice_num}. {options[choice_num]}")
+    return options
+
+
+def make_choice(options, customer, video):
+    valid_choices = options.keys()
+    choice = None
+
+    while choice not in valid_choices:
+        print("What would you like to do? Select 1 for all available options!")
+        choice = input("Make your selection using the option number: ")
+
+    if choice in ['3','4'] and customer.selected_customer == None:
+        print("You must select a customer before updating or deleting them.")
+        print("Let's select a Customer!")
+        choice = "5"
+
+    if choice in ['9','10'] and video.selected_video == None:
+        print("You must select a video before updating or deleting it.")
+        print("Let's select a video!")
+        choice = "7"
+    
+    return choice
+
+def run_cli(play=True):
+
+    #initialize customer
+    customer = CustomerOps("https://retro-video-store-api.herokuapp.com", None)
+    video = VideoOps ("https://retro-video-store-api.herokuapp.com", None)
+    # print choices
+    options = list_options()
+
+    while play==True:
+
+        # get input and validate
+        choice = make_choice(options, customer, video)
+
+        # customer.print_selected()
+        if choice =="1":
+            list_options()
+
+        elif choice=='2':
+            print("Great! Let's add a customer.")
+            name=input("First and last name? ")
+            phone=input("What is your phone number? ")
+            postal_code =input("Enter your 5 digit zip code!")
+            response = customer.add_customer(name=name, phone=phone, postal_code=postal_code)
+            print("New customer:", response["customer"])
+
+        elif choice=='3':
+            print(f"Great! Let's update the customer info: {customer.selected_customer}")
+            name=input("First and last name? ")
+            phone=input("What is your phone number? ")
+            postal_code =input("Enter your 5 digit zip code!")
+            response = customer.update_customer(name=name, phone=phone, postal_code=postal_code)
+            print(response)
+            # print("Updated customer:", response["customer"])
+
+        elif choice=='4':
+            customer.delete_customer()
+            print("Customer has been deleted.")
+
+            print(customer.list_customers())
+
+        elif choice=='5':
+            print("List of all customers!")
+            for cust in customer.list_all_customers():
+                print(cust)
+
+        elif choice=='6':
+            id = input("Select the customer id!")
+            if id.isnumeric():
+                id = int(id)
+                customer.get_customer(id=id)
```

Comment:
Great job checking if `id` is numeric, but what happens if the user types in an invalid id? Adding an else statement to handle that edge case would be nice.